manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml

```diff
@@ -0,0 +1,26 @@
+PackageIdentifier: ManifoldSoftware.Manifold-Release-8
+PackageVersion: 8.0.31
+PackageName: Manifold Release 8 Ultimate Edition
```

Comment:
```suggestion
PackageName: Manifold Systems
```

Reply:
"Manifold Release 8" and "Manifold Release 9" This is how Manifold Software calls their products. I'd hesitate to remove "Release" from the name. Though, with latest update, every edition of Manifold 8 is now Ultimate Edition, so perhaps
`PackageName: Manifold Release 8`

 https://manifold.net/info/manifold8.shtml
https://manifold.net/info/manifold9.shtml

Reply:
Well... Thats the developers problem, both `Manifold Release 8` and `9` gets installed as `Manifold Systems` in `A`dd or `R`emove `P`rograms which WinGet uses to match the program.

Reply:
![image](https://user-images.githubusercontent.com/1541866/130761216-ccc7caa4-5c03-41e5-a015-923c141e2c86.png)
I suggest `PackageName: Manifold System 8` ( and `PackageName: Manifold System 9` for the other)

Reply:
The problem with commenting on 2 places about the same thing https://github.com/microsoft/winget-pkgs/pull/25779#discussion_r695549892 😁 Go ahead with the change :)